src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```
@@ -305,4 +305,12 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
+  <Target Name="AfterBuild">
+    <!-- Drop a file in the artifacts directory containing the ANCM version number -->
+    <ItemGroup>
+      <VersionFileContents Include="$(AspNetCoreModuleVersionMajor).$(AspNetCoreMinorVersion).$(AssemblyBuild).$(AspNetCorePatchVersion)" />
+      <VersionFileContents Include="$(CommitHash)" />
```

```suggestion
      <VersionFileContents Include="$(SourceRevisionId)" />
```
CommitHash is going to be deprecated. SourceRevisionId is a well-known, built-in property in Microsoft.Common.targets available after the  InitializeSourceControlInformation target . Example usage:

https://github.com/aspnet/AspNetCore/blob/78bc2a10b9ea617269527f4091903471a299e197/eng/targets/CSharp.Common.targets#L9-L17

---

Awesome, I'm glad this is in `Microsoft.Common.targets` now!